src/test/app/claims/models/claim.ts

```diff
@@ -148,4 +154,78 @@ describe('Claim', () => {
       expect(claim.defendantOffer).to.be.eq(undefined)
     })
   })
+
+  describe('claimStatus', () => {
```

Comment:
Please add more cases, eg. more time requested + eligible for ccj -> make sure that when claim matches both, "the more important" status is returned.

Reply:
Done 👍